src/test/java/seedu/address/logic/commands/BlacklistCommandTest.java

```diff
@@ -0,0 +1,46 @@
+// package seedu.address.logic.commands;
```

Comment:
The first test case looks like it's still a valid one, could it be worth keeping it?